frontend/src/app/patients/Components/ManageEmails.test.tsx

```diff
@@ -1,5 +1,6 @@
+/* eslint testing-library/no-unnecessary-act:0 */
```

Comment:
since we are wrapping our userEvent calls with `act` should we remove this? 🤔

Reply:
actually this is to disable the eslint rule that says not to wrap anything in act. Basically the react testing library already wraps its logic in the act method so in theory developers should not have to do it manually hence eslint raising check errors `unnecesary-act` when seeing act wrapping. However, we are not using `fireEvent` to trigger events but `userEvent` so the wrapping is not properly done and we do need to manually add it or we start seeing warning left and right polluting our logs... so that is why I disabled the eslint rule for this file.

Reply:
ahhhh ok! thank you for this breakdown!! :D  -- please ignore my previous comment ^_^